src/components/buttons/Button.tsx

```diff
@@ -28,7 +29,7 @@ import { BORDER_RADIUS } from '~/style/globalStyle'
 export interface ButtonProps extends PressableProps {
   title?: string
   type?: 'primary' | 'secondary' | 'transparent' | 'tint'
-  variant?: 'default' | 'contrast' | 'accent' | 'valid' | 'alert'
+  variant?: 'default' | 'contrast' | 'accent' | 'valid' | 'alert' | 'highlight'
```

Comment:
Good initiative! But with this new prop we should maybe open an issue to one day get rid of the standalone `HighlightButton` which may be confusing... WDYT?

Reply:
Agreed

Reply:
https://github.com/alephium/mobile-wallet/issues/253